pkgs/tools/text/ansifilter/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, pkgs }:
+let version = "1.15";
+    pkgsha = "65dc20cc1a03d4feba990f830186404c90462d599e5f4b37610d4d822d67aec4";
+in with pkgs; stdenv.mkDerivation {
+  name = "ansifilter";
+  buildInputs = [
+    stdenv gcc gnumake
```

Comment:
`stdenv`, `gcc`, and `gnumake` are redundant build inputs (the point of stdenv is to provide an environment where essential build commands are available).